pymongoexplain/utils.py

```diff
@@ -30,12 +30,6 @@ def convert_to_camelcase(d, exclude_keys=[]):
         if "_" in key and key[0] != "_":
             new_key = key.split("_")[0] + ''.join(
                 [i.capitalize() for i in key.split("_")[1:]])
-        if isinstance(d[key], list):
-            ret[new_key] = [convert_to_camelcase(
-                i, exclude_keys=exclude_keys) for i in d[key]]
-        elif isinstance(d[key], dict):
-            ret[new_key] = convert_to_camelcase(d[key],
-                                                     exclude_keys=exclude_keys)
```

Comment:
Great! I think we can remove the `exclude_keys` argument now too.

Reply:
Done.